ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -681,9 +681,9 @@ ENABLE_USED_PASSWORDS_CHECK = True
 {% endif %}
 
 {% if localsettings.get('ENIKSHAY_PRIVATE_API_USERS') %}
-ENIKSHAY_PRIVATE_API_USERS = localsettings.get('ENIKSHAY_PRIVATE_API_USERS')
+ENIKSHAY_PRIVATE_API_USERS = {{ localsettings.get('ENIKSHAY_PRIVATE_API_USERS') }}
 {% endif %}
 
 {% if localsettings.get('ENIKSHAY_PRIVATE_API_PASSWORD') %}
-ENIKSHAY_PRIVATE_API_PASSWORD = localsettings.get('ENIKSHAY_PRIVATE_API_PASSWORD')
+ENIKSHAY_PRIVATE_API_PASSWORD = {{ localsettings.get('ENIKSHAY_PRIVATE_API_PASSWORD') }}
```

Comment:
this needs to be in quotes if it's a string `'{{ localsettings.get('ENIKSHAY_PRIVATE_API_PASSWORD') }}'`